kafka-client/src/main/java/dev/responsive/kafka/internal/utils/Utils.java

```diff
@@ -122,4 +122,8 @@ public static String extractThreadIdFromThreadName(final String threadName) {
     return threadName;
   }
 
+  public static long millisToSeconds(final long millis) {
+    return millis / 1000L;
+  }
```

Comment:
nit we should prefer `TimeUnit.MILLISECONDS.toSeconds(millis)` and just remove this method (it handles more edge cases that I don't believe we can ever hit but it's good to have)